nexmo/__init__.py

```diff
@@ -184,30 +212,42 @@ def submit_sms_conversion(self, message_id, delivered=True, timestamp=None):
         return self.post(self.api_host, "/conversions/sms", params)
 
     def send_event_alert_message(self, params=None, **kwargs):
+        """ redirects to rest.nexmo.com/sc/us/alert/json with params if it exists """
         return self.post(self.host, "/sc/us/alert/json", params or kwargs)
 
     def send_marketing_message(self, params=None, **kwargs):
+        """ redirects to rest.nexmo.com/sc/us/marketing/json with params if it exists """
         return self.post(self.host, "/sc/us/marketing/json", params or kwargs)
 
     def get_event_alert_numbers(self):
+        """ redirects to rest.nexmo.com/sc/us/alert/opt-in/query/json """
         return self.get(self.host, "/sc/us/alert/opt-in/query/json")
 
     def resubscribe_event_alert_number(self, params=None, **kwargs):
+        """ redirects to rest.nexmo.com/sc/us/alert/opt-in/manage/json """
         return self.post(self.host, "/sc/us/alert/opt-in/manage/json", params or kwargs)
 
     def initiate_call(self, params=None, **kwargs):
+        """ redirects to rest.nexmo.com/call/json with params if it exists """
         return self.post(self.host, "/call/json", params or kwargs)
 
     def initiate_tts_call(self, params=None, **kwargs):
+        """ redirects to api.nexmo.com/tts/json with params if it exists """
         return self.post(self.api_host, "/tts/json", params or kwargs)
 
     def initiate_tts_prompt_call(self, params=None, **kwargs):
+        """ redirects to api.nexmo.com/tts-prompt/json with params if it exists """
         return self.post(self.api_host, "/tts-prompt/json", params or kwargs)
 
     def start_verification(self, params=None, **kwargs):
+        """ redirects to api.nexmo.com/verify/json with params if it exists """
         return self.post(self.api_host, "/verify/json", params or kwargs)
 
     def send_verification_request(self, params=None, **kwargs):
+        """ 
+        deprecated feature. warning appears
```

Comment:
This docstring contains notes that you've derived from reading the code. It's not really the same as reference documentation.

May I recommend that you read [PEP-257](https://www.python.org/dev/peps/pep-0257/) for guidance on writing docstrings. Also the following documentation branch shows the quality of documentation we're hoping to achieve: https://github.com/Nexmo/nexmo-python/blob/code-docs/nexmo/__init__.py#L63-L108